mining.md

```diff
@@ -107,26 +107,54 @@ Chain selection is a crucial component of how the Filecoin blockchain works. Eve
 
 ### Block Validation
 
-The block structure and serialization is detailed in  [the datastructures spec](data-structures.md#block). Check there for details on fields and types.
+The block structure and serialization is detailed in [the datastructures spec](data-structures.md#block). Check there for details on fields and types.
 
 In order to validate a block coming in from the network at round `N` was well mined a miner must do the following:
 
-1. Validate that `BlockSig` validates with the miner's public key (`miner.PublicKey`) over the block content.
-2. Validate `ParentWeight`
-   - Each of the blocks in the block's `ParentTipset` must have the same `ParentWeight`.
-   - The new block's `ParentWeight` must have been properly calculated using the [chain weighting function](./expected-consensus#chain-weighting).
-3. Validate `StateRoot`
-   - In order to do this, a miner must ensure that applying the block's messages to the `ParentState` (not included in block header) appropriately yields the `StateRoot` and the receipts in the `ReceiptRoot`.
-4. Validate that the tickets in the `Tickets` array are valid, thereby proving appropriate delay for this block creation.
-   - Ensure that the new ticket was generated from the smallest ticket in the block's parent TipSet (at round `N-1`).
-   - Recompute the new ticket, using the miner's public key, ensuring it was computed appropriately.
-5. In a case where the block contains multiple values in the `Tickets` array
-   - Ensure that all tickets are signed by the same key.
-   - Ensure that each ticket was used to generate the next, with the first generated from the smallest ticket in the Parent tipset (at `N-1`, as above).
```

Comment:
either the text of 4 and 5 (ie tickets verif details) should be included as a comment here, or `verifyTickets` should be written out. It isn't defined in EC currently. Though it can be derived from it, we should be explicit here.

Reply:
VerifyTickets is below, in the ticket validation section.

Reply:
perfect. apologies